gammapy/datasets/simulate.py

```diff
@@ -303,7 +300,6 @@ def event_list_meta(dataset, observation):
         meta["NMCIDS"] = len(dataset.models)
 
         # Necessary for DataStore, but they should be ALT and AZ instead!
-        meta["ALTITUDE"] = observation.aeff.meta["CBD50001"][7:-4]
         meta["ALT_PNT"] = observation.aeff.meta["CBD50001"][7:-4]
```

Comment:
There is still a test fail related to this: https://github.com/gammapy/gammapy/runs/4126240358?check_suite_focus=true#step:9:2148

I think the solution here is to create the information from the observer position like so:
```python3
meta = {
    "TELESCOP": "CTA",
    "INSTRUME": "Southern Array"
}

if meta["TELESCOP"] == "CTA" and meta["INSTRUME"] == "Southern Array":
    loc = observatory_locations["cta_south"]
elif meta["TELESCOP"] == "CTA" and meta["INSTRUME"] == "Northern Array":
    loc = observatory_locations["cta_south"]
else:
    pass

# this is not really correct but maybe OK for now
altz_frame = AltAz(obstime=dataset.gti.time_start, location=loc)

coord_altaz = observation.pointing_radec.transform_to(altaz_frame)
```